ansible/secrets.enc.template

```diff
@@ -0,0 +1,4 @@
+TRELLO_API_KEY: <DELIBERATELY MISSING>
```

Comment:
In theory, if this is actually being encrypted then it's reasonable to include here, but  I definitely won't argue against a defense-in-depth approach of not posting it!
Edit: ah, I notice you literally say the same in your Readme 😄